src/Tools/Source/CompilerGeneratorTools/DeployCompilerGeneratorToolsRuntime/project.json

```diff
@@ -2,8 +2,8 @@
   "dependencies": {
     "Microsoft.NETCore.Portable.Compatibility": "1.0.1-beta-23504",
     "Microsoft.NETCore.Platforms": "1.0.1-beta-23504",
```

Comment:
These other dependencies will need to be updated as well, especially the Platforms one.